plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/OpenShiftServerUtils.java

```diff
@@ -297,6 +322,42 @@ public static String getPodPath(IServerAttributes attributes) {
 		return getProjectAttribute(ATTR_POD_PATH, null, getDeployProject(attributes));
 	}
 	
+
+	/**
+	 * Creates an {@link RSync}
+	 * @param server the {@link IServer} on which the {@code rsync} operation will be performed
+	 * @return the {@link RSync} to be used to execute the command.
+	 * @throws CoreException
+	 */
+	public static RSync createRSync(final IServer server) throws CoreException {
```

Comment:
`OCBinaryOperation` already has a `createRSync(IServer)` method. Shouldn't you re-use it if it has the same behaviour ?